### ISSUE TITLE
reorderingSingleHash fails

### ISSUE THREAD

**Issue by mrMakaronka** [data-streams/data-streams#8]
[reorderingSingleHash.txt](https://github.com/data-streams/data-streams/files/1065770/reorderingSingleHash.txt)

**Comment by marnikitta** [comment]
1c1
< Actual
---
> Expected
247d246
< [5955106624674911148, 4311608066345280133, 1694700219858867495, -3895109916546361993],
551d549
< [-3039124234266745498, 5955106624674911148, 4311608066345280133, 1694700219858867495],
621a620
> [-3039124234266745498, 5955106624674911148, -7875412322565894446, 4311608066345280133],
666a666
> [-7875412322565894446, 4311608066345280133, 1694700219858867495, -3895109916546361993],
679a680
> [5937401334208193644, -3039124234266745498, 5955106624674911148, -7875412322565894446],
905d905
< [5937401334208193644, -3039124234266745498, 5955106624674911148, 4311608066345280133],
934a935
> [5955106624674911148, -7875412322565894446, 4311608066345280133, 1694700219858867495],

**Comment by marnikitta** [comment]
-78 появился из неоткуда

**Comment by mrMakaronka** [comment]
[reorderingSingleHash-2.txt](https://github.com/data-streams/data-streams/files/1065803/reorderingSingleHash-2.txt)

**Comment by mrMakaronka** [comment]
[full_log.txt](https://github.com/data-streams/data-streams/files/1065809/full_log.txt)

**Comment by marnikitta** [comment]
```
2c2
< Actual   :[5404997405243787776, -1393891633583620431, 3008912362058444677, 1835214581069490196, 7749711512032494165, 5934698359568496122, 6147559547584117058],
---
> Expected :[5404997405243787776, -1393891633583620431, 3008912362058444677, 1835214581069490196, 7749711512032494165, 5934698359568496122, 6147559547584117058],
469c469
<  [-8026846873268049860, -6303816040687659497, 5270999777087709493, -8748331363195382623, -7828798931423620050, -599867274932125906, -3539600637613101681],
---
>  [5270999777087709493, -8748331363195382623, -7828798931423620050, -599867274932125906, 1777528827791953817, -3539600637613101681, -3564545758675126954],
476d475
<  [-8748331363195382623, -7828798931423620050, -599867274932125906, -3539600637613101681, -3564545758675126954, -2837295052393202101, -1106246392983545002],
586d584
<  [5270999777087709493, -8748331363195382623, -7828798931423620050, -599867274932125906, -3539600637613101681, -3564545758675126954, -2837295052393202101],
602a601
>  [-6303816040687659497, 5270999777087709493, -8748331363195382623, -7828798931423620050, -599867274932125906, 1777528827791953817, -3539600637613101681],
622d620
<  [-7828798931423620050, -599867274932125906, -3539600637613101681, -3564545758675126954, -2837295052393202101, -1106246392983545002, -7221241905877065242],
706a705
>  [-7828798931423620050, -599867274932125906, 1777528827791953817, -3539600637613101681, -3564545758675126954, -2837295052393202101, -1106246392983545002],
731d729
<  [-599867274932125906, -3539600637613101681, -3564545758675126954, -2837295052393202101, -1106246392983545002, -7221241905877065242, -2856934510614998399],
778d775
<  [-6303816040687659497, 5270999777087709493, -8748331363195382623, -7828798931423620050, -599867274932125906, -3539600637613101681, -3564545758675126954],
800a798
>  [-599867274932125906, 1777528827791953817, -3539600637613101681, -3564545758675126954, -2837295052393202101, -1106246392983545002, -7221241905877065242],
866a865
>  [1777528827791953817, -3539600637613101681, -3564545758675126954, -2837295052393202101, -1106246392983545002, -7221241905877065242, -2856934510614998399],
945a945
>  [-8748331363195382623, -7828798931423620050, -599867274932125906, 1777528827791953817, -3539600637613101681, -3564545758675126954, -2837295052393202101],
982a983
>  [-8026846873268049860, -6303816040687659497, 5270999777087709493, -8748331363195382623, -7828798931423620050, -599867274932125906, 1777528827791953817],
```

**Comment by mrMakaronka** [comment]
Fixed: aa89917
